table/table_test.cc

```diff
@@ -2604,16 +2604,44 @@ TEST_F(GeneralTableTest, ApproximateOffsetOfCompressed) {
 }
 
 TEST_F(HarnessTest, Randomized0) {
-  // part 1 out of 2
+  // part 1 out of 4
   const size_t part = 1;
-  const size_t total = 2;
+  const size_t total = 4;
   RandomizedHarnessTest(part, total);
 }
 
 TEST_F(HarnessTest, Randomized1) {
-  // part 2 out of 2
+  // part 2 out of 4
   const size_t part = 2;
-  const size_t total = 2;
+  const size_t total = 4;
```

Comment:
Should not this be 8?

Reply:
randomized0 does first quarter of work, randomized1 does second quarter.
The remaining half are split 4 ways into randomized5 through randomized 8. 
In other words, [0, 1] is split into [0, 0.25, 0.5, 0.625, 0.75, 0.875, 1] segments

Reply:
Oh I see. How about keeping "total = 8" and can call RandomizedHarnessTest(part, total) twice when you want it to do twice the work, like
RandomizedHarnessTest(part, total);
RandomizedHarnessTest(part + 1, total);